introduction.md

```diff
@@ -0,0 +1,3 @@
+Name:Karthik
```

Comment:
JavaScript

On Tue, Aug 18, 2015 at 11:46 PM, Code School Kiddo <
notifications@github.com> wrote:

> In introduction.md
> <https://github.com/deadlyvipers/dojo_rules/pull/6443#discussion_r37333419>
> :
>
> > @@ -0,0 +1,3 @@
> > +Name:Karthik
>
> Looks good! Could you also please mention your favorite Code School path
> in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or
> Electives.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/deadlyvipers/dojo_rules/pull/6443/files#r37333419>.
>



-- 

Thanks & Regards,

P.Karthik
Trainee
Vensi Inc.